Documentation/WritingReST/CheatSheet.rst

```diff
@@ -337,6 +337,21 @@ Source::
       $a = 'hello';
 
 
+Placeholders
+------------
+
+Use the *Backus-Naur form* `<placeholder-name>` for placeholders in code and
```

Comment:
```suggestion
Use the `Backus-Naur form<https://en.wikipedia.org/wiki/Backus-Naur_form>`__ like `<placeholder-name>` for placeholders in code and
```

Reply:
I try to reduce external links. One of my guidelines is to keep the reader on the page or at least in the documentation or TYOP3 universe, and do not try him to leave. If he really wants to look up the name, (s)he should do a copy&paste of the name.

Reply:
And I try to add as much explanation as possible, using external links if possible. This way the reader has a curated resource to look at. He is free to skip the link if not necessary.

If you prefer to keep people in the TYPO3 docs then we should add a short explanation what BNF is. This way one can continue reading without being puzzled or having to read an external explanation which would interrupt the reading flow.

Reply:
> He is free to skip the link if not necessary.

Yes, but he gets distracted with every linked word. On the other hand i would not know which part of the BNF explanation would be required to further understand the way we want to highlight the placeholder.

I don't mind too much, i see that there two ways to approach linking of external sources, so i would suggest to discuss it in the TYPO3 Documentation Team and come up with a guideline.

Reply:
Suggestion to simplify this: drop the term _Backus-Naur form_ altogether. What is not mentioned does not need any explanation. It's just a syntax we chose to indicate placeholders/variables, that it's based on BNF is actually not really important.

Reply:
I would like to keep it in the text, as it is an explanation of why we have chosen it (as it is following some standard) and would like to discuss the external topic link in the team.